### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/states-of-matter#237]
Due to https://github.com/phetsims/atomic-interactions/issues/72 it was decided a short memory test should be done on the recently published sim.  Results:
Start: 27.2 MB
1 min: 56.6 MB
2 min: 74.9 MB
3 min: 94.2 MB
4 min: 119 MB
5 min: 137 MB

**Comment by jbphet** [comment]
I tested a RequireJS version (1.1 release branch) with the fixes in the commit above, and the results look good.  Here's a table:


| Time (minutes)  | Memory Usage (MB)  |
|---|---|
| 0  | 30.2 |
| 1  | 37.5  |
| 2  | 39.3  |
| 3  | 40.3 |
| 4  | 40.7  |
| 5  | 41.0 |
| 6  | 41.2  |
| 7  | 41.4  |
| 8  | 41.7  |
| 9  | 42.0  |
| 10  | 42.2  |

The memory is increasing, but not by much, and the total is well within the expected range for a sim of this size and complexity.  It's also more than a 3x reduction in memory usage at the 5 minutes mark.  And, finally, I looked at several comparisons of the heap snapshots captured during the testing, and there were no leaks of SOM-specific objects.  All increases in memory usage were from things like "array" and "compiled code".   Here is a screenshot from a comparison of two snapshots:

![image](https://user-images.githubusercontent.com/3082831/48649455-e24c4d00-e9af-11e8-906f-d6f41e510867.png)

The bottom line on this is that I think the leak is fixed, and I should do a maintenance release on it.

**Comment by jbphet** [comment]
I should do a maintenance release on States of Matter 1.1 to publish this fix.

**Comment by jbphet** [comment]
Redeployment complete.  I tested the built version before deployment, and the memory number looked good (a little better than in https://github.com/phetsims/states-of-matter/issues/237#issuecomment-439541860 actually).  Closing.
